.eslintrc

```diff
@@ -0,0 +1,62 @@
+{
+  "env": {
+    "node": true
+  },
+  "globals": {},
+  "rules": {
+    "no-bitwise": 0,
+    "camelcase": 0,
+    "curly": 0,
+    "eqeqeq": 0,
+    "guard-for-in": 0,
+    "no-extend-native": 0,
+    "wrap-iife": [
+        2,
+      "any"
+    ],
+    "no-use-before-define": 0,
+    "new-cap": 0,
+    "no-caller": 2,
+    "no-empty": 2,
+    "no-new": 2,
+    "no-plusplus": 2,
+    "no-undef": 2,
+    "no-unused-vars": [2, {"vars": "all", "args": "none"}],
+    "strict": 0,
+    "max-params": [
+        2,
+        99
+    ],
+    "max-depth": [
+        2,
+        99
+    ],
+    "complexity": [
+        2,
+        99
+    ],
```

Comment:
Let's remove `max-params`, `max-depth`, and `complexity`. I've used arbitrarily large values for such options in the past and exceeded the limits (in test suites with lots of assertions).